packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js

```diff
@@ -31,7 +31,7 @@ export const styles = theme => {
         opacity: 0.38,
       },
     },
-    /* Styles applied to the root element if `expanded={true}`. */
+    /* Styles applied to the root element, `content` and `expandIcon` if `expanded={true}`. */
```

Comment:
Those are CSS classes. Could you describe the semantics instead of the actual CSS classes?

Reply:
@eps1lon Sure thing!